server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeTSIServiceImpl.java

```diff
@@ -141,7 +142,13 @@ public DataNodeTSIServiceImpl() {
   @Override
   public TSOpenSessionResp openSession(TSOpenSessionReq req) throws TException {
     IoTDBConstant.ClientVersion clientVersion = parseClientVersion(req);
-    TSStatus loginStatus = AuthorityChecker.checkUser(req.username, req.password);
+    TSStatus loginStatus;
+    try {
+      loginStatus = AuthorityChecker.checkUser(req.username, req.password);
+    } catch (ConfigNodeConnectionException e) {
+      TSStatus tsStatus = RpcUtils.getStatus(TSStatusCode.INTERNAL_SERVER_ERROR, e.getMessage());
```

Comment:
use authentication error?

Reply:
Maybe we need to add a new TSStatusCode.